using_packages/conanfile_txt.rst

```diff
@@ -49,11 +49,9 @@ Where:
     string; it does not have to be a number, so, for example, it could indicate if this is a "develop" or "master" version.
     Packages can be overwritten, so it is also OK to have packages like "nightly" or "weekly", that
     are regenerated periodically.
-  - ``pocoproject`` is the owner of this package version. It is basically a namespace that allows
-    different users to have their own packages for the same library with the same name, and
-    interchange them. So, for example, you can upload a certain library under your own user
-    name, and later the same packages can be uploaded, without modifications, to another
-    official group or company username.
+  - ``pocoproject`` is the owner of this package. It is basically a namespace that allows different users to have their own packages for
+    the same library with the same name. For example, you can upload a certain library under your own user/organization name without
```

Comment:
I'd say it is not very clear. The interesting part of using your own username is that it allows to do changes to the packages.

Reply:
I think that the audience for this documentation is likely to understand the significance of namespacing in this context, and we don't even need what comes after the first sentence.

Reply:
Removed last sentence then